aws/cloudformation/components/alarms.yml.erb

```diff
@@ -1,25 +1,20 @@
-  DaemonStorageUtilizationAlarm:
-    Type: "AWS::CloudWatch::Alarm"
+  Critical: {Type: AWS::SNS::Topic}
+  Warning: {Type: AWS::SNS::Topic}
+<%{Warning: 70, Critical: 90}.each do |name, threshold| -%>
+  DiskUsed<%=name%>:
+    Type: AWS::CloudWatch::Alarm
     Properties:
-      AlarmActions:
-        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-<%= rack_env?(:production) ? 'Urgent' : 'LowPriority'%>"
-      AlarmDescription: Send page if daemon storage utilization exceeds 80% for an hour
-      AlarmName: <%="#{stack_name}_daemon_high_storage_utilization" %>
-      ComparisonOperator: GreaterThanThreshold
-      Dimensions:
-        - Name: Filesystem
-          Value: '/dev/xvda1'
-        - Name: MountPath
-          Value: '/'
-        - Name: InstanceId
-          Value: <%= daemon_instance_id || "!Ref #{daemon}" %>
-      EvaluationPeriods: 60
-      MetricName: DiskSpaceUtilization
-      Namespace: 'System/Linux'
+      AlarmActions: [!Ref '<%=name%>']
+      AlarmDescription: Max disk usage in <%=environment%> env exceeds <%=threshold%>%
```

Comment:
This is not terribly important, but the Zoomers may not know what a "disk" :) I recognize that the underlying metric published by CWAgent uses that term, but can we use Storage Utilization in the Resource name and the Alarm Description?

Reply:
I'd prefer to leave it as 'disk', both for naming consistency with the underlying CloudWatch metric being published and also to distinguish the local disk-based filesystem from other network-attached storage (memcached, database, etc).

Reply:
A bit of Googling tells me 'storage' pretty consistently refers to persistent filesystems and not database services, so that doesn't seem relevant. I do still prefer keeping the alarm's name aligned with the underlying CloudWatch metric though.